# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieCommons.h
# Commit: fe6e63a2f914
# Full Hash: fe6e63a2f9141d9eeec41bfde52cc34b33c170fe
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-10-09 14:54 +0000
# Description:
#   Bug 1992536 - Use cookie creation time instead or PR_Now() to validate the expiration time, r=valentin,cookie-reviewers
# ==============================================================================

diff -r 0147060f88c2 -r fe6e63a2f914 netwerk/cookie/CookieCommons.h
--- a/netwerk/cookie/CookieCommons.h	Thu Oct 09 14:36:53 2025 +0000
+++ b/netwerk/cookie/CookieCommons.h	Thu Oct 09 14:54:32 2025 +0000
@@ -183,9 +183,9 @@
   // assuming no leading dots are present.
   static bool IsSubdomainOf(const nsACString& a, const nsACString& b);
 
-  // Returns the current time in msecs using a nsIChannel, which corresponds to
+  // Returns the current time in USecs using a nsIChannel, which corresponds to
   // the response start time.
-  static int64_t GetCurrentTimeFromChannel(nsIChannel* aChannel);
+  static int64_t GetCurrentTimeInUSecFromChannel(nsIChannel* aChannel);
 };
 
 }  // namespace net